package/yast2-geo-cluster.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Sep 14 15:27:03 UTC 2017 - nwang@suse.com
+
+- fate#323863. support autoyast.
+- Version 3.3.0
```

Comment:
for master we decide to increase to 4.0.0. see http://yastgithubio.readthedocs.io/en/latest/versioning/